timesketch/api/v1/resources/sigma.py

```diff
@@ -47,53 +58,201 @@ def get(self):
         sigma_rules = []
 
         try:
-            sigma_rules = ts_sigma_lib.get_all_sigma_rules()
+            sigma_rules_results = SigmaRule.query.filter_by()
```

Comment:
You want to get all rules, correct? Then you can do
`SigmaRule.query.all()`

Reply:
done